src/client/opamSolution.ml

```diff
@@ -1291,23 +1294,23 @@ let install_depexts ?(force_depext=false) ?(confirm=true) t packages =
                                  f status.OpamSysPkg.s_available }
               sys_map
           | None -> sys_map)
-        packages
+        new_packages
         (Lazy.force t.sys_packages)
     in
-    { t with sys_packages = lazy sys_packages }
+    Some ({ t with sys_packages = lazy sys_packages })
```

Comment:
```suggestion
    Some { t with sys_packages = lazy sys_packages }
```

Reply:
Thanks! Is there a formatter that I can run on my changes at all?

Reply:
no we're not using ocamlformat. But i don't think it's needed anyway, your formatting looks fine otherwise

Reply:
As for formatters in general, we use ocp-indent but it would not have caught that as it is only a simple formatter, not a re-format tool